source/clear-linux/tutorials/kubernetes/kubernetes-bp.rst

```diff
@@ -0,0 +1,98 @@
+.. _kubernetes-bp:
+
+Kubernetes Best Practices on |CL|
```

Comment:
Please update title to not use "Clear Linux OS", also to use sentence case - eg. update to "Kubernetes best practices"